### DISCUSSION TITLE
Listening for when the server is stopping

### DISCUSSION THREAD

**Discussion by Braayy** [Minestom/Minestom#1067] [Help] [answered]
Is there a way of listening to when the server is beeing stopped? The `Runtime#addShutdownHook(Thread)` only works when the server has been "SIGINTinted"

**Comment by GreatWyrm** [comment, marked as answer]
You will want to use [SchedulerManager#buildShutdownTask()](https://javadoc.minestom.net/net/minestom/server/timer/SchedulerManager.html#buildShutdownTask(java.lang.Runnable)).
